src/io/flutter/editor/FlutterPubspecNotificationProvider.java

```diff
@@ -55,13 +55,12 @@ public EditorNotificationPanel createNotificationPanel(@NotNull VirtualFile file
       return null;
     }
 
-    final Module module = ModuleUtilCore.findModuleForFile(file, project);
-    if (module == null || !FlutterModuleUtils.usesFlutter(module)) {
+    // Check that this pubspec file declares flutter
+    if (!PubRoot.declaresFlutter(file)) {
       return null;
     }
 
-    final FlutterSdk sdk = FlutterSdk.getFlutterSdk(project);
-    if (sdk == null) {
+    if(FlutterSdk.getFlutterSdk(project) == null) {
```

Comment:
Perhaps format?  I'd expect a space after the `if`...

Reply:
Done